our-work/software/soma.yml

```diff
@@ -0,0 +1,14 @@
+title: SOMA pain & recovery
+description: A mobile app that combines symptom tracking with neuroscientific assessments and tailored interventions. SOMA is designed to directly support individuals on their journeys to overcome pain.
+languages:
+  - typescript
+  - react native
+tags:
+  - mobile
+people:
+  - Bradford Roarr
+    github_user: broarr
+  - Ellen Duong
+    github_user: eldu
+groups:
+  - PEAC Lab
```

Comment:
I think use `peak-lab` instead. Spaces may complicate the matching we'll try to do with these groups later

Reply:
I changed `PEAC Lab` to `peac-lab`